### ISSUE TITLE
Null Pointer exception dragging an object

### ISSUE THREAD

**Issue by SteveGreatApe** [Samsung/GearVRf#1716]
I hit this exception using GVRCursorController startDrag(GVRSceneObject dragMe) and stopDrag(). It seems to happen if you very quickly grab and release the object.

It looks to me likes the cause is that some of the StartDrag/StopDrag processing is run using runOnGlThreadPostRender. This doesn't guarantee that Runnables added for processing are run in the same order they are added. Therefore the StopDrag Runnable can be run before the StartDrag Runnable, in which case mDragMe gets set to null first, then accessed in the StartDrag runnable causing the Exception.
```
FATAL EXCEPTION: GLThread 7039
Process: com.greatape.avrgallery, PID: 4375
	   java.lang.NullPointerException: Attempt to invoke virtual method 'org.gearvrf.GVRTransform org.gearvrf.GVRSceneObject.getTransform()' on a null object reference
     at org.gearvrf.GVRCursorController$1.run(GVRCursorController.java:428)
     at org.gearvrf.GVRViewManager.afterDrawEyes(GVRViewManager.java:508)
     at org.gearvrf.OvrViewManager.afterDrawEyes(OvrViewManager.java:312)
     at org.gearvrf.OvrViewManager.onDrawFrame(OvrViewManager.java:302)
     at org.gearvrf.OvrVrapiActivityHandler$8.onDrawFrame(OvrVrapiActivityHandler.java:449)
     at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1562)
     at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1262)
```

**Comment by bohuang3d** [comment]
Is the dragMe passed to startDrag() non-null?

Can you try moving the null checks in startDrag() to within the synchronized block in mStartDrag()?

**Comment by NolaDonato** [comment]
I will do that now.

**Comment by NolaDonato** [comment]
Pull request #1720 attempts to fix this problem. Please let me know if it does.

**Comment by NolaDonato** [comment]
Are you still having this problem or can I close this issue?

**Comment by SteveGreatApe** [comment]
I've actually changed the way my code works so I can't test this any more.

**Comment by NolaDonato** [comment]
Fixed by #1720
